src/sentry/api/endpoints/assistant.py

```diff
@@ -41,26 +53,37 @@ def get(self, request):
         seen_ids = set(
             AssistantActivity.objects.filter(user=request.user).values_list("guide_id", flat=True)
         )
-        for k, v in guides.items():
-            v["seen"] = v["id"] in seen_ids
+
+        for key, value in six.iteritems(guides):
+            value["seen"] = value["id"] in seen_ids
+
+        if "v2" in request.GET:
+            guides = [
+                {"guide": key, "seen": value["seen"] in seen_ids}
```

Comment:
Won't `value["seen"]` be a boolean here, and so always be `False`? Probably worth covering this as part of tests.

Reply:
Ahhh 
```
>>> True in {1}
True
```
Anyway, fixed in the tests. Thanks!

Reply:
Damn... TIL